docs/guides/react-native.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: React Native
+description: Using Jotai in React Native
+nav: 3.5
+---
+
+Jotai atoms can be used in React Native applications with very few changes except for the use of persistence.
+
+Jotai has an [atomWithStorage function in the utils bundle](../api/utils.mdx#atom-with-storage) for persistance that supports persisting state in `sessionStorage`, `localStorage`, `AsyncStorage`, or the URL hash.
+
+Because react-native only supports AsyncStorage, you need to tell Jotai to use AsyncStorage instead of the default.
+
+Then, depending on what you are storing, you may need to take additional steps.  By default, AsyncStorage stores strings.  So, if you are simply storing a string, just passing AsyncStorage in is enough.
+
+## A simple pattern with AsyncStorage
+
+```js
+export const strAtom = atomWithStorage('string', undefined, AsyncStorage)
+```
+
+If you want to store objects and booleans, however, there are a series of helper functions that make this easier.
+
+## Storing non-string items in AsyncStorage
+
+First, create a storage item using the createJSONStorage function.  This automatically stringifies and parses objects before writing to AsyncStorage.
+
+```js
+export const storage = createJSONStorage(() => AsyncStorage)
+```
+
+Now you can pass an object into the store just by passing storage as the storage type.
+
+```js
+export const storage = createJSONStorage(() => AsyncStorage)
+export const objectAtom = atomWithStorage('object', undefined, storage)
+```
+
+## Storing booleans in AsyncStorage
+
+To store a boolean, there is an additional helper function atomWithToggleAndStorage.
+
+```js
+export const storage = createJSONStorage(() => AsyncStorage)
+export const booleanAtom = atomWithToggleAndStorage('boolean', false, storage)
```

Comment:
This atom helper is only described in an advanced recipe. I'm a little bit concerned to have it here. At least we need a link to the recipe.

Reply:
I have left this in for now but have added a link to the recipe.  I added some text throughout so you may want to review the new commit again, and I've updated to meet your new numbering.